### PR TITLE
Create Github Workflow File to automatically build release artifacts using Github Actions

### LINKED ISSUE #15: Automatically build release artifacts using Github Actions
Since the last release, nobody updated the pre-built version of the UI in the github releases. Could someone have a look at this and how to prevent this in the future?

### PR DESCRIPTION
**Pull request by mittulmandhan**
# Description
This pull request includes a github workflow file that does the following actions on every release:
1.) Check out the repo on release tag
2.) Print the tag name
3.) Install dependencies in vitrivr-ng
4.) Run build vitrivr-ng
~~5.) Run tests with coverage~~(commented)
5.) Set an environment variable to get the tag name
6.) archive the artifacts(vitrivr-ng/dist/)
7.) Add artifacts to the release assets

**Fixes #15**

# Are there any open issues?
No

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (wiki, README, Website)
- [x] I have updated the pre-built binary on the release page (for major changes)

### CODE REVIEW

**Review by silvanheller — APPROVED** [review]
This looks good to me, i'll wait for the approval of @ppanopticon before merging

**Comment by ppanopticon** [comment]
Looks good to me.

**Review by ppanopticon — APPROVED** [review]

**Pull request merged**
